sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -3,12 +3,20 @@
 ## 4.0.0-beta.5 (Unreleased)
 
 ### Features Added
+- Added support for address type field value.
+- Added support for vertex coordinates with model `Point` to represent polygon vertices in `boundingPolygon` property.
```

Comment:
CC @HarshaNalluru we should ensure we have similar entries for everything listed here. I know we have caption/footnote, address fields, etc. But I think we are missing paragraphs, removal of entities, and renaming languageCode -> locale.